content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -394,6 +394,43 @@ NAME              PARENTREF
 2001:db8:1:2::a   services/kube-system/kube-dns
 ```
 
+This new feature also allow users to dynamically define the `service-cluster-ip-range` using
+ServiceCIDR objects. During bootstrap, a default ServiceCIDR object named `kubernetes` is created
+from the flags:
```

Comment:
"from the flags" ... can be more specific.

Reply:
updated